en/python_introduction/README.md

```diff
@@ -683,7 +683,7 @@ if 3 > 2:
     print('It works!')
 ```
 
-Notice how we've indented the next line of code by 4 spaces? We need to do this so Python knows what code to run if the result is true. You can do one space, but nearly all Python programmers do 4 to make things look neat. A single `tab` will also count as 4 spaces.
+Notice how we've indented the next line of code by 4 spaces? We need to do this so Python knows what code to run if the result is true. You can do one space, but nearly all Python programmers do 4 to make things look neat. A single Tab will also count as 4 spaces as long as your text editor is set to do so. When you made your choice, don't change it! If you already indented with 4 spaces, make any future indentation with 4 spaces, too - otherwise you may run into problems.
```

Comment:
Not sure the 'as long long as your editor` part makes sense, but I think this does improve it.

Reply:
> Not sure the 'as long long as your editor` part makes sense,

There's at least 3 aspects here:

1. Will the tab **key** insert a tab **character** or a number of space characters?
   - If the latter, how many?
2. If a tab character is inserted, how long will it appear in the editor?
3. If tabs and spaces are mixed, how will Python interpret them?

Dunno how much of that the instructions here should cover, though.

Reply:
That was my consideration as well. Too many technical details can be confusing and actually worse than the full story. Most important part is not mixing spaces and tabs and that was added now.